src/trajectory.jl

```diff
@@ -104,6 +105,7 @@ Slice sampler carried during the building of the tree.
 It contains the slice variable and the number of acceptable condidates in the tree.
 """
 struct SliceTreeSampler{F<:AbstractFloat} <: AbstractTreeSampler
+    zcand   ::  PhasePoint
     logu    ::  F     # slice variable in log space
```

Comment:
Perhaps `logu` ==> `ℓu` for consistency.

Reply:
Good idea.